src/components/AssignmentTexter/ContactToolbar.jsx

```diff
@@ -128,12 +128,10 @@ const ContactToolbar = function ContactToolbar(props) {
         title={global.ASSIGNMENT_CONTACTS_SIDEBAR ? "Toggle Contact List" : ""}
       >
         <IconButton
+          aria-label="Toggle Contact List"
           className={css(styles.contactToolbarIconButton)}
-          onClick={() => {
-            global.ASSIGNMENT_CONTACTS_SIDEBAR
```

Comment:
What is this `global. ASSIGNMENT_CONTACTS_SIDEBAR` ? and why do we not need to check it anymore?

Reply:
`ASSIGNMENT_CONTACTS_SIDEBAR` is an environment variable. Previously, if `ASSIGNMENT_CONTACTS_SIDEBAR` is set, the "Toogle Contact List" button will run the function `props.toggleContactList()` when it's clicked, else it does nothing. I added the condition property `disabled={global.ASSIGNMENT_CONTACTS_SIDEBAR ? false : true}` to disable the "Toggle Contact List" button if `ASSIGNMENT_CONTACTS_SIDEBAR` isn't set. I did this to prevent confusion because the JAWS screen reader reads it as a clickable button even though it actually doesn't do anything when clicked. If the `disabled` property to set to `true`, the `onClick` method automatically doesn't run, so no need to check for the `global.ASSIGNMENT_CONTACTS_SIDEBAR` condition anymore.